Orange/widgets/visualize/owboxplot.py

```diff
@@ -271,7 +271,7 @@ def __init__(self):
         self.mainArea.layout().addWidget(self.box_view)
 
         e = gui.hBox(self.mainArea, addSpace=False)
```

Comment:
I suppose if you do just `gui.hBox(self.mainArea, addSpace=False)` Travis will be happy. 🤷‍♀